src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categoriesList: [],
+  status: false,
+  msg: '',
+};
+/* eslint no-param-reassign: "error" */
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      if (state.status === false) {
+        state.msg = 'under construction';
+      }
+    },
+  },
```

Comment:
- [ ] As stated in the project required in the categories page there should be a button called `check status` and when the button is been clicked, there should be a text display above or below the button stating `Under Construction`. But currently in your project and i cant see any button in the categories page, rather what i see is a text `Hello Categories`.  Kindly work on this part too